py/desitarget/mtl.py

```diff
@@ -412,6 +412,93 @@ def get_mtl_ledger_format():
     return ff
 
 
+def check_archiving(zcatdir=None, mtldir=None):
```

Comment:
as suggested above, what about adding additional `survey="main"` and a `obscon="DARK"` arguments?

Reply:
Addressed, see my response to your first comment, above.